janusgraph-cql/pom.xml

```diff
@@ -16,8 +16,10 @@
         <test.byteorderedpartitioner>byteorderedpartitioner</test.byteorderedpartitioner>
         <test.murmur>murmur</test.murmur>
         <test.murmur-ssl>murmur-ssl</test.murmur-ssl>
-        <test.jvm.opts>-Xms256m -Xmx1280m -ea -XX:+HeapDumpOnOutOfMemoryError -javaagent:${com.github.jbellis:jamm:jar}</test.jvm.opts>
+        <test.extra.jvm.opts>-javaagent:${com.github.jbellis:jamm:jar}</test.extra.jvm.opts>
```

Comment:
How about instead:
```
<test.javaagent.opts>-javaagent:${com.github.jbellis:jamm:jar}</test.javaagent.opts>
<test.extra.jvm.opts>${test.javaagent.opts}</test.extra.jvm.opts>
```
to avoid repeating later on when overriding:
```
<test.extra.jvm.opts>${jacoco.opts} ${test.javaagent.opts}</test.extra.jvm.opts>
```